CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+Thank you for your interest in contributing! We value your contributions. 🙏
+
+This guide will discuss how the team handles [Commits](#commits), [Pull Requests](#pull-requests), [Releases](#releases), the [Changelog](#changelog), and [Response time](#response-time).
+
+**Note:** We won't force external contributors to follow this verbatim, but following these guidelines definitely helps us in accepting your contributions.
+
+> [!NOTE]
+> Please tag @DeveloperRelations if you have any questions.
```

Comment:
I'm not sure that external developers can tag entire teams btw.

Reply:
ah bummer. ok i will remove this note from all of these